ext/fileinfo/tests/finfo_open_001.phpt

```diff
@@ -45,3 +64,10 @@ Warning: finfo_open(%sinexistent): Failed to open stream: No such file or direct
 
 Warning: finfo_open(): Failed to load magic database at "%sinexistent" in %s on line %d
 bool(false)
+object(finfo)#%d (%d) {
+}
+object(finfo)#%d (%d) {
+}
+finfo::__construct(%s123): Failed to open stream: No such file or directory
+finfo::__construct(%s1): Failed to open stream: No such file or directory
+finfo::__construct(%sinexistent): Failed to open stream: No such file or directory
```

Comment:
Test file should end with a newline (same remark for the other three files).

Reply:
thanks for the review, I am a newbie and I appreciate it.

during the couples of days I will o my best to send a batch of QA related changes, hopefully increasing covergae